website/templates/layout.jade

```diff
@@ -54,7 +54,10 @@ html(lang='en')
                   -   linkCssClass = 'active';
                   - }
                   li(class=linkCssClass)
-                    a(href=link.src, target=link.target) #{link.text}
+                    if link.src
+                      a(href=link.src, target=link.target) #{link.text}
+                    else
+                      div(style="padding: 10px 15px;") !{link.text}
```

Comment:
could you please move the styling to the .css file (website/contents/assets/main.css) if there's no matching bootstrap class?